andela/persephone-ah-backend
# #166816093 Setup tests environment

#### What does this PR do?
- Setup Tests Environment


#### Description of Task to be completed?
- Delete config folder
- Delete models folder
- Delete routes folder
- Install Mocha Chai and Chai-http
- Refactor entry file to remove all mongodb related connections
- Write endpoint for base url
- Write test for base url endpoint to ensure all tests are passing
- delivers [#166816093]

#### How should this be manually tested?
- Clone the repository using ```git clone https://github.com/andela/persephone-ah-backend.git .``` in your terminal
- run ```npm install``` on your terminal
- run ```npm run test``` to see test results

#### What are the relevant pivotal tracker stories?
- PT Story link - [#166816093](https://www.pivotaltracker.com/n/projects/2353116/stories/166816093)

#### Screenshots (if appropriate)
- N/A

#### Questions:
- N/A
